pullrequests/PollPR.py

```diff
@@ -0,0 +1,721 @@
+"""
+# Aiming to be used with Hydra (Nixos) builder for Ghaf project https://github.com/tiiuae/ghaf
+
+# Activates Hydra (Ghaf docker based Hydra build system) to build new open Pull Requests
+#(for the main repo under observations)
+#
+# Rebuilds still open (and previously built) PR if there has been new changes for that PR
+# Uses Ghaf build tools from https://github.com/tiiuae/ci-public
+
+# Can be used in service mode (polling frequently changes) or cherrypicking some wished open PR
+# Keeps internal recoed which open PRs or changed PRs have been allready processed
+# (given Hydra build commands ok)
+#
+#
+#
+# Author mika.nokka1@gmail.com 13.6.2023
```

Comment:
I was basically forced to remove my original author messages, so forcing you to remove your author messages.